build.sh

```diff
@@ -725,6 +725,69 @@ menu_option_23() {
   echo "Done."
 }
 
+
+menu_option_24() {
+  INPUT_PATH="$SCRIPT_PATH"/builds/childs/template
+  echo -e "\nCONFIRM: Build a Ubuntu Server 22.04 LTS child from VM Template Template for VMware vSphere?"
```

Comment:
"Template Template"

Reply:
Done in commit 675af11a3a1d537610faac594556c4dea0ab05d4